src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/JSON.txt

```diff
@@ -0,0 +1,79 @@
+// (UTC-08:00) Pacific Time (US & Canada)
+#SETUP: TimeZoneInfo("Pacific Standard Time")
+#SETUP: OptionSetTestSetup
+
+>> JSON(Blank())
+"null"
+
+>> JSON("hello")
+"""hello"""
+
+>> JSON(1.23)
+"1.23"
+
+>> JSON(0)
+"0"
+
+>> JSON(1)
+"1"
+
+>> JSON(-1)
+"-1"
+
+>> JSON(true)
+"true"
+
+>> JSON(false)
+"false"
+
+>> JSON(RGBA(10, 30, 50, 0.75))
+"""#0a1e32bf"""
+
+>> JSON(Color.Turquoise)
+"""#40e0d0ff"""
+
+>> JSON(OptionSet.Option2)
+"""option_2"""
+
+>> JSON(TimeUnit.Hours)
+"""hours"""
+
+// This only works because we're in Pacific Time
+// If we would be in UTC+N time zone, the result would be "2022-08-06" as we convert the date to UTC
```

Comment:
This is a bug, we should fix it prior to committing this change to main

Reply:
Separated PR will come first

Reply:
I believe the bug in on PA Client side

PA Client (with French TimeZone)

Text(Date(2022, 11, 23), DateTimeFormat.UTC)
-> 2022-11-22T23:00:00.000Z

JSON(DateTimeValue("2022-11-23"))
-> "2022-11-22T23:00:00.000Z"

JSON(Date(2022, 11, 23))
-> "2022-11-23" 
// Should be 2022-11-22 to be consistent

Reply:
Following our discussion, I fixed this